GoogleForCreators/web-stories-wp
# Quick Action Menu: `Change background color` opens the page background panel if it is collapsed

## Context



## Summary

Clicking the `Change background color` quick action when the background is selected opens the page background panel (if it is closed).

## Relevant Technical Choices

n/a

## To-do

n/a

## User-facing changes

|Before|After|
|--|--|
|![no-open-clicky-clicky](https://user-images.githubusercontent.com/22185279/121225769-16760a80-c847-11eb-9e34-53f7e63b2f5c.gif)|![open-clicky-clicky](https://user-images.githubusercontent.com/22185279/121225377-b2ebdd00-c846-11eb-9aca-ac51790b162a.gif)|

## Testing Instructions

1. Go to editor
2. Click background element
3. Collapse the page background panel
4. Click the `Change background color` quick action
5. Page background panel should open

### QA






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #7866
